### ISSUE TITLE
Image does not fill the entire window

### ISSUE THREAD

**Issue by SlyAceZeta** [shusaura85/notparadoxlauncher#32]
Is it just me, or is the image not properly sized to fill the entire launcher window? This seems like it shouldn't be the case...

![image](https://user-images.githubusercontent.com/11371137/151486717-dc4156f0-257a-4b91-a27f-7650e78f8621.png)

**Comment by shusaura85** [comment]
Yeah, it is supposed to fill the entire window. I'm guessing it's due to the scaling in display settings being high enough that the image used for the background is not big enough for it, and the image is not set to expand. It won't affect the functionality of it.  
  
If I manage to find time to update it, I'll fix it for the next version.  
  
If you use [Resource Hacker](http://www.angusj.com/resourcehacker/) you can fix it using these steps:  
1. open NPL "launcher.exe" using Resource Hacker  
2. In the tree view on the left, open the "RCData" section and click on "TNOTLAUNCHERWINDOW: 0" entry under RCData  
3. In the right side, go to line 16085, the line should be "**Stretch = True**"  
4. Add a new line that contains "**Stretch = True**" and ensure it has the same number of spaces as the previous line  
5. Compile the resource (Menu "Action" -> Compile, or F5, or the green arrow on the toolbar)  
6. Save the file (it will automatically create a backup file)  
  
If you did it correctly when you launch the game, the launcher should show the image on the entire window

**Comment by SlyAceZeta** [comment]
> It won't affect the functionality of it.

Oh, I know, I've been using the launcher like this for months, just never thought to report it 😅

Thank you very much for those instructions, Resource Hacker fixed it perfectly!

![image](https://user-images.githubusercontent.com/11371137/151626634-a0214e38-1e79-4f3c-a908-8dc5559d17c3.png)
